assets/src/components/routeLadder.tsx

```diff
@@ -63,25 +67,49 @@ export const Header = ({
   return (
     <Card className="c-route-ladder__header">
       <Card.Body>
-        <div className="c-route-ladder__header__action-container">
+        <div
+          className={joinClasses([
+            "c-route-ladder__dropdown",
+            hasAlert && "non-skate-alert",
```

Comment:
I'd love to get @firestack's take on this, because she knows BEM better than I do, but I _think_ that we want to name `non-skate-alert something like `c-route-ladder__dropdown--non-skate-alert` to make it clear that it's a modifier of `c-route-ladder__dropdown`.

```suggestion
            hasAlert && "c-route-ladder__dropdown--non-skate-alert",
```